include/deal.II/lac/lapack_full_matrix.h

```diff
@@ -444,6 +444,12 @@ class LAPACKFullMatrix : public TransposeTable<number>
    */
   number reciprocal_condition_number(const number l1_norm) const;
 
+  /**
+   * Same as above but for triangular matrices. The matrix has to have
+   * the LAPACKSupport::Property set, see set_property().
+   */
+  number reciprocal_condition_number() const;
```

Comment:
You say "same as above", but what norm is this then computed for?

Does the property have to be set to something specific?